packages/scroller/src/vaadin-scroller.js

```diff
@@ -63,6 +63,16 @@ class Scroller extends ElementMixin(ThemableMixin(PolymerElement)) {
       scrollDirection: {
         type: String,
         reflectToAttribute: true
+      },
+
+      /**
+       * Indicates whether the element can be focused and where it participates in sequential keyboard navigation.
+       * @protected
+       */
+      tabindex: {
```

Comment:
Let's consider using `TabindexMixin` instead (which would also require us to use `DisabledMixin`).
I guess that's fine because users might want to be able to disable focusing the scroller.